consutil/main.py

```diff
@@ -21,66 +21,80 @@ def consutil():
 
     if os.geteuid() != 0:
         click.echo("Root privileges are required for this operation")
-        sys.exit(1)
+        sys.exit(ERR_CMD)
 
 # 'show' subcommand
 @consutil.command()
 def show():
-    """Show all /dev/ttyUSB lines and their info"""
-    devices = getAllDevices()
-    busyDevices = getBusyDevices()
+    """Show all lines and their info"""
+    lines = getAllLines()
+    busyLines = getBusyLines()
 
-    header = ["Line", "Actual/Configured Baud", "PID", "Start Time", "Device"]
+    # sort lines for table rendering
+    lines.sort(key=lambda dev: int(dev[LINE_KEY]))
+
+    # set table header style
+    header = ["Line",  "Baud",   "PID",   "Start Time", "Device"]
     body = []
-    for device in devices:
-        lineNum = device[LINE_KEY]
+    for line in lines:
+        # configured information
+        lineNum = line[LINE_KEY]
+        baud = '-' if BAUD_KEY not in line else line[BAUD_KEY]
+        remoteDevice = '-' if DEVICE_KEY not in line else line[DEVICE_KEY]
+
+        # runtime information
         busy = " "
         pid = ""
         date = ""
-        remoteDevice = '-' if DEVICE_KEY not in device else device[DEVICE_KEY]
-        if lineNum in busyDevices:
-            pid, date = busyDevices[lineNum]
+        if lineNum in busyLines:
+            pid, date = busyLines[lineNum]
             busy = "*"
-        actBaud, confBaud, _ = getConnectionInfo(lineNum)
-        # repeated "~" will be replaced by spaces - hacky way to align the "/"s
-        baud = "{}/{}{}".format(actBaud, confBaud, "~"*(15-len(confBaud)))
         body.append([busy+lineNum, baud, pid, date, remoteDevice])
-        
-    # replace repeated "~" with spaces - hacky way to align the "/"s
-    click.echo(tabulate(body, header, stralign="right").replace('~', ' ')) 
+    click.echo(tabulate(body, header, stralign='right')) 
 
 # 'clear' subcommand
 @consutil.command()
-@click.argument('linenum')
-def clear(linenum):
+@click.argument('target')
+def clear(target):
     """Clear preexisting connection to line"""
-    checkDevice(linenum)
-    linenum = str(linenum)
+    targetLine = getLine(target)
+    if not targetLine:
+        click.echo("Target [{}] does not exist".format(linenum))
+        sys.exit(ERR_DEV)
+    lineNumber = targetLine[LINE_KEY]
 
-    busyDevices = getBusyDevices()
-    if linenum in busyDevices:
-        pid, _ = busyDevices[linenum]
+    busyLines = getBusyLines()
+    if lineNumber in busyLines:
+        pid, _ = busyLines[lineNumber]
         cmd = "sudo kill -SIGTERM " + pid
```

Comment:
I am wondering if we need to add some more check there before sending that kill signal.

What if we have a pid logged in db but somehow process quit didn't clear the db. However, some critical process took the PID. Blindly sending the kill could be dangerous. @jleveque what do you think?

Reply:
This comments doesn't block this PR. If change needed, the change can come with a separate PR.

Reply:
@yxieca: Adding a check to ensure the PID is still assigned to the expected process is probably a good idea to add in a future PR. It definitely wouldn't hurt.

Reply:
Yes, totally agree.

Currently, it is safe to clear it by the PID because the PID were extract from PS command.
The check will be necessary after I introduced STATE_DB.
I will add checks in future PR. Thanks for comments